tor_core/config.py

```diff
@@ -251,6 +252,10 @@ def heartbeat_port(self):
 except OSError:
     Config.slack_api_url = os.environ.get('SLACK_API_URL', None)
 
+try:
+    Config.sentry_api_url = open('sentry.key').readline().strip()
```

Comment:
Why are we naming the file `sentry.key` if it's the sentry api url?

I know, the Slack url is the same way. I'm arguing that it should be named something else, or maybe having a key-value store config file (ini? toml?) if we're reading from the filesystem first.

Reply:
...I don't really have a reason. I like the idea of having separate files because that way we don't have to worry about an INI file not having the right fields and versioning them -- we just have to say "Is the file there? Yes / no" and move on. Other than that, it's just "because that's how we've been doing it". Do you think it's worth changing?

Reply:
Eh, I have a feeling we'll tackle it at a later point with maybe a bot-config file or something similar. Not a problem for today since it follows convention, but I wanted to bring it up since I noticed it this time.